R/dm.R

```diff
@@ -659,7 +673,44 @@ compute.zoomed_dm <- function(x, ...) {
 src_tbls.dm <- function(x) {
   # The x argument here is a dm object
   dm <- x
-  names(dm_get_tables(dm))
+  # FIXME: should this be forbidden for `zoomed_dm`? and should we not just take the names from `def`?
```

Comment:
```suggestion
  check_not_zoomed(x)
```

and move before the assignment. `names(...)` is fine here.